readme.md

```diff
@@ -12,3 +12,26 @@ A fun side project by some lovely folks at [Tighten Co.](http://tighten.co/).
 
 * [Api documentation](doc/api.md)
 * Chat about it on freenode IRC channel `#symposium`
+
+### Requirements
+
+* PHP >= 5.5.9
+* PHP [mcrypt extension](http://php.net/manual/en/book.mcrypt.php)
+* A [supported relational database](http://laravel.com/docs/5.1/database#introduction) and corresponding PHP extension
+* [Composer](https://getcomposer.org/download/)
+
+### Installation
+
+1. [Fork this repository](https://help.github.com/articles/fork-a-repo/)
+2. [Install dependencies](https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies)
+3. Copy [`.env.sample`](https://github.com/tightenco/symposium/blob/master/.env.example) to `.env` and modify its contents to reflect your local environment. In particular, be sure to set `DB_CONNECTION` to a database connection name from [configuration](https://github.com/tightenco/symposium/blob/master/config/database.php) and any applicable related values.
```

Comment:
It's actually `.env.example`. I'd rather we not have the explicit note about `DB_CONNECTION`, but you know what? I'll just merge these and then edit them. Thanks!